resources/build/trigger-builds.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+#
+# Tell TeamCity to trigger new builds
+#
+
+function triggerBuilds() {
+  local base=`git branch --show-current`
+  if [[ $base == master ]]; then
+    local TEAMCITY_BUILDTYPES=( KeymanAndroid_Build Keyman_iOS_Master KeymanLinux_Master KeymanMac_Master Keyman_Build Keymanweb_Build )
+    local TEAMCITY_VCS_ID=HttpsGithubComKeymanappKeyman
+  elif [[ $base == beta ]]; then
+    local TEAMCITY_BUILDTYPES=( KeymanAndroid_Beta Keyman_iOS_Beta KeymanLinux_Beta KeymanMac_Beta KeymanDesktop_Beta Keymanweb_Beta )
+    local TEAMCITY_VCS_ID=Keyman_KeymanappKeymanBeta
+  else
+    exit 1
+  fi
+
+  for TEAMCITY_BUILDTYPE in "${TEAMCITY_BUILDTYPES[@]}"; do
```

Comment:
So, if my understanding of this is right, we auto-trigger all builds whenever there's a version-number increase?  No filtering of unchanged platforms?

Reply:
Correct: I think if we have synchronized version numbers it simplifies everyone's life when every version is available for all platforms. Yeah, that means a new iOS build for Apple to review every day but that's their choice!

For stable, the builds will be rare enough that (a) extra builds is not a big concern, and (b) consistent version numbers is even more useful.